3.0/README.md

```diff
@@ -39,28 +39,76 @@ Implementations MUST treat unknown keys as if they weren't present. However, imp
 
 ## 3.1 `attribution`
 
-REQUIRED|OPTIONAL (description of dependents/dependencies)
+OPTIONAL. Default: null.
 
-**Description:**
+Contains an attribution to be displayed when the map is shown to a user. Implementations MAY decide to treat this as HTML or literal text. For security reasons, make absolutely sure that this field can't be abused as a vector for XSS or beacon tracking.
 
-**Example:**
+```JSON
+{
+  "attribution": "<a href='http://openstreetmap.org'>OSM contributors</a>"
+}
+```
 
 ## 3.2 `bounds`
 ## 3.3 `center`
 ## 3.4 `data`
 ## 3.5 `description`
+
+OPTIONAL. Default: null.
+
+A text description of the tileset. The description can contain any legal character. Implementations SHOULD NOT interpret the description as HTML.
+
+```JSON
+{
+  "description": "We are volcanoes. When we women offer our experience as our truth, as human truth, all the maps change. There are new mountains."
+}
+```
+
 ## 3.6 `grids`
 ## 3.7 `legend`
 ## 3.8 `maxzoom`
 ## 3.9 `minzoom`
+
 ## 3.10 `name`
+
+OPTIONAL. Default: null.
+
+A name describing the tileset. The name can contain any legal character. Implementations SHOULD NOT interpret the name as HTML.
+
+```JSON
+{
+  "name": "Earthsea v2"
+}
+```
+
 ## 3.11 `scheme`
 ## 3.12 `template`
 ## 3.13 `tilejson`
```

Comment:
@GretaCB in the previous version we mention the `tilejson` value being semver, but our new `3.0` spec only defines major.minor and no patch. Should we rephrase this?

Reply:
@mapsam Good eye. I'm thinking we should change this and make it super specific when to bump minor and when to bump major. For example, fixing insignificant typos/spelling errors wouldnt need any kind of version bump. But if a change *affects implementation in anyway* (like the spelling of a tilejson property), that would require a major bump.

Reply:
Nice - I agree. We also mention when major.minor in the [CONTRIBUTING.md](https://github.com/mapbox/tilejson-spec/blob/master/CONTRIBUTING.md#proposing-changes) which we wrote recently, which appears to nicely match your example.